frontend/packages/otr/src/interfaces/clerkInterpreter.ts

```diff
@@ -0,0 +1,34 @@
+import { APIResponseStatus } from 'shared/enums';
+import { WithId, WithVersion } from 'shared/interfaces';
+
+import { Qualification, QualificationResponse } from 'interfaces/qualification';
+
+export interface ClerkInterpreter extends WithId, WithVersion {
+  deleted: boolean;
+  identityNumber: string;
+  firstName: string;
+  lastName: string;
+  email: string;
+  permissionToPublishEmail: boolean;
+  permissionToPublishPhone: boolean;
+  permissionToPublishOtherContactInfo: boolean;
+  qualifications: Array<Qualification>;
+  regions: Array<string>;
```

Comment:
Nämä voi halutessa vaihtaa päittäin kun siinä formaatissa swaggerissäkin

Reply:
Aakkosjärjestys.